tests/unit/test.config.js

```diff
@@ -259,6 +266,23 @@ describe('config', () => {
       const newArgv = applyConfigToArgv({argv, configObject, defaultValues});
       assert.strictEqual(newArgv.sourceDir, '/configured/directory');
     });
+
+    it('throws an error when an option is invalid', () => {
+      const {argv, defaultValues} = makeArgv({
+        globalOpt: {
+          'source-dir': {
+            type: 'string',
+            demand: false,
+          },
+        },
+      });
+      const configObject = {
+        foo: 'fake/value/',
+      };
+      assert.throws(() => {
+        applyConfigToArgv({argv, configObject, defaultValues});
+      }, UsageError, /Please fix your config and try again/);
```

Comment:
This regex should make sure that the option name was specified in the error message